.github/workflows/ci.yml

```diff
@@ -58,23 +58,42 @@ jobs:
       if: matrix.os == 'ubuntu-latest'
       run: |
         python -m pip install --upgrade pip setuptools wheel
+        # mltool installs
+        python -m pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+        python -m pip install xgboost
+        python -m pip install tritonclient[grpc,http]
+        # install checked out coffea
         python -m pip install -q -e '.[dev,parsl,dask,spark]'
         python -m pip list
         java -version
     - name: Install dependencies (MacOS)
       if: matrix.os == 'macOS-latest'
       run: |
         python -m pip install --upgrade pip setuptools wheel
+        # mltool installs
+        python -m pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+        python -m pip install xgboost
+        # install checked out coffea
         python -m pip install -q -e '.[dev,dask,spark]'
         python -m pip list
         java -version
     - name: Install dependencies (Windows)
       if: matrix.os == 'windows-latest'
       run: |
         python -m pip install --upgrade pip setuptools wheel
+        # mltool installs
+        python -m pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
+        python -m pip install xgboost
+        # install checked out coffea
         python -m pip install -q -e '.[dev,dask]'
         python -m pip list
         java -version
+
+    - name: Start triton server with example model
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        docker run  --rm   -p 8001:8001 -p 8002:8002  -v$(realpath ./coffea/tests/samples/triton_models_test):/models nvcr.io/nvidia/tritonserver:23.04-py3 tritonserver --model-repository=/models
```

Comment:
Final testing issue! For triton testing, we will need to spin up a parallel docker instance. Throwing the command I used for local testing on my laptop here while I look up the docs on how you do this properly in GitHub workflows.